pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[tool.isort]
+profile = "black"
```

Comment:
`pyproject.toml` is the modern python project configuration file / format.
This config block just makes `isort` and `black` lint tools play nice with each other since they overlap somewhat in responsibility.